.github/workflows/nightly_snapshot.yaml

```diff
@@ -368,7 +368,7 @@ jobs:
                   cp -a ../target/doc/* $output_path/docs/rust/
 
                   # Fix up link to Slint language documentation
-                  sed -i "s!https://slint.dev/releases/.*/docs/!../../!" $output_path/docs/rust/slint/*.html
+                  sed -i "s!https://releases.slint.dev/.*/docs/!../../!" $output_path/docs/rust/slint/*.html
```

Comment:
The link are still using slint.dev in the docs, so either leave it as it was before, or repleace all other occurrences in the repo